src/sub-blocks/BackgroundCard/BackgroundCard.tsx

```diff
@@ -46,16 +68,18 @@ const BackgroundCard = (props: BackgroundCardProps) => {
                     style={{backgroundColor}}
                 />
                 <Content
+                    titleId={titleId}
                     title={title}
                     text={text}
                     additionalInfo={additionalInfo}
                     size="s"
                     theme={cardTheme}
-                    links={links}
-                    buttons={buttons}
+                    links={areControlsInFooter ? undefined : links}
+                    buttons={areControlsInFooter ? undefined : buttons}
                     colSizes={{all: 12, md: 12}}
                 />
             </CardBase.Content>
+            {footerControls}
```

Comment:
```
<ContentControls ...>
  {(children) => renderCardFooterControlsContainer(children)}
</ContentControls>
```

Reply:
Unfortunately, this won't work within CardBase.